daisy_workflows/image_build/debian/build_fai.py

```diff
@@ -49,6 +49,71 @@ def mycopytree(src, dst):
     return dst
 
 
+# get number of bytes to offset when mounting the raw disk image file
+def get_mount_offset(fdisk_output):
+    single_str = fdisk_output.stdout.decode('utf-8')
+    subprocess.run(['echo', 'single string'])
+    subprocess.run(['echo', single_str])
```

Comment:
why are you piping `echo` here? why not using `logging`? (applies everywhere i.e. lines 58, 60 etc)

Reply:
changed to use logging